### PR TITLE
Initial repo setup

### PR DESCRIPTION
**Pull request by samaloney**
Some things probably need to be updated

### CODE REVIEW

**Comment by vit1-irk** [comment]
Why Python 3.10 and not 3.12 or so?

**Comment by samaloney** [comment]
Following [SPEC 0](https://scientific-python.org/specs/spec-0000/) and [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html) this would just be the minimum supported version so idea would be it work on python3.10 and greater including 3.11 and 3.12

**Review by vit1-irk — APPROVED** [review]

**Comment by vit1-irk** [comment]
Let Gelu merge it, he has the write access

**Comment by samaloney** [comment]
Yea there maybe other stuff you want changed also can use this as an example of tho the process works.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  "wheel",
+  "extension-helpers",
+  "numpy>=1.25",
+  "cython"
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyampp"
+description = "Automatic Model Suff"
```
Comment (samaloney) [inline pyproject.toml:14]: Like I think you want a better description but I don't know what that would be

**Pull request merged**